### DISCUSSION TITLE
logstransform processor not included in package

### DISCUSSION THREAD

**Discussion by ghost** [open-telemetry/opentelemetry-collector-contrib#17851] [Q&A] [answered]
Hey Guys,

In my latest collector setup I could really use the [logstransform processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/logstransformprocessor). Although, according to the release changes, it should be part of the current distribution but I am not able to use it inside the docker container _otel/opentelemetry-collector-contrib:latest_. 

Since I am new to OpenTelemetry I am not sure if the missing processor is a bug or intended. I just saw that it is not included in [this](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/internal/components/components.go) go code, which might be the reason for the absence inside the container...

Can you help me out here?
Best,
Fabian

**Comment by atoulme** [comment, marked as answer]
I don't see logstransform listed in https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/internal/components/components.go indeed. It needs to be added to become available, please open an issue asking it to be added.

**Reply by ghost** [reply]
Thank you, I will.
